.github/workflows/ci-cd-renv.yml

```
@@ -1,12 +1,21 @@
 # Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
 # Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 
-# Triggered on push and pull request events
-on: [push, pull_request]
-
 # Name of the workflow => usethis::use_github_actions_badge("CI-CD-renv")
 name: CI-CD-renv
 
+# Triggered on push and pull request events
+# workflow_dispatch allows you to run this workflow manually from the Actions tab
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  # ...on a regular schedule for checking existing links turning broken; this is
+  # set to run each 1st of the month to make sure Actions caches are
+  # preserved (they expire every 7 days) and available for future runs
+  schedule:
+    - cron:  '0 0 1 * *'
```

Rephrased and consistent comments. Note that the `schedule`d runs are only relevant for ShinyCICD and will not end up in `techguides`.

```suggestion
on:
  # Triggered on push and pull request events
  push:
  pull_request:
  # Allow manual runs from the Actions tab
  workflow_dispatch:
  # Monthly runs on the 1st day of the month at midnight
  # (specific for ShinyCICD maintenance)
  schedule:
    - cron:  '0 0 1 * *'
```

---

The same applies to `ci-cd.yml`

---

Updated comments on both files.